modules/tabs-v2/widgets/tabs-v2.php

```diff
@@ -0,0 +1,432 @@
+<?php
+namespace Elementor\Modules\TabsV2\Widgets;
+
+use Elementor\Controls_Manager;
+use Elementor\Core\Kits\Documents\Tabs\Global_Colors;
+use Elementor\Core\Kits\Documents\Tabs\Global_Typography;
+use Elementor\Group_Control_Text_Shadow;
+use Elementor\Group_Control_Typography;
+use Elementor\Modules\NestedElements\Base\Widget_Nested_Base;
+use Elementor\Modules\NestedElements\Controls\Control_Nested_Repeater;
+use Elementor\Plugin;
+use Elementor\Repeater;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+class TabsV2 extends Widget_Nested_Base {
+
+	public function get_name() {
+		return 'tabs-v2';
+	}
+
+	public function get_title() {
+		return esc_html__( 'Nested Tabs', 'elementor' );
+	}
+
+	public function get_icon() {
+		return 'eicon-tabs';
+	}
+
+	public function get_keywords() {
+		return [ 'nested', 'tabs', 'accordion', 'toggle' ];
+	}
+
+	protected function get_default_children_elements() {
+		return [
+			[
+				'elType' => 'container',
+				'settings' => [
+					'_title' => __( 'Tab #1', 'elementor' ),
+				],
+			],
+			[
+				'elType' => 'container',
+				'settings' => [
+					'_title' => __( 'Tab #2', 'elementor' ),
+				],
+			],
+		];
+	}
+
+	protected function get_default_repeater_title_setting_key() {
+		return 'tab_title';
+	}
+
+	protected function get_default_children_title() {
+		return esc_html__( 'Tab #%d', 'elementor' );
+	}
```

Comment:
Try to use `get_default_children_title` instead of hard-coded values in `get_default_children_elements`

Reply:
We came to the conclusion that we prefer to use `get_default_children_elements` default settings instead of using 2 hooks to handle the issue.